tests/functional/base_pages/hub.page.js

```diff
@@ -18,11 +18,11 @@ class HubPage extends BasePage {
   }
 
   errorHeader() {
-    return "#main-content > div.panel.panel--error.u-mb-s > div.panel__header > div";
+    return '[data-qa="error-header"]';
   }
 
   errorNumber(number = 1) {
-    return `[data-qa="error-body"] ul > li:nth-child(${number}) > a`;
+    return `[data-qa="list-item-${number}"]`;
```

Comment:
There are a number of selectors that use `nth-child`. We should be consistent in how we reference elements that can be repeated.

Reply:
So do you think I should change all the nth-child selectors to use data-qa attributes too?

Reply:
I think so, using an example:
```
tbody:nth-child(${instance}) td:last-child a[data-qa='change-item-link']
```

would be something like:
```
a[data-qa='change-item-link-${instance}']
```

Which is simpler and less fragile, as it's not dependent on that element being in a `td`, in a `tbody`.